programs/install/Install.cpp

```diff
@@ -214,7 +214,7 @@ int mainInstall(int argc, char ** argv)
         desc.add_options()
             ("help,h", "produce help message")
             ("prefix", po::value<std::string>()->default_value("/"), "prefix for all paths")
-            ("binary-path", po::value<std::string>()->default_value("usr/bin"), "where to install binaries")
+            ("binary-path", po::value<std::string>()->default_value("usr/local/bin"), "where to install binaries")
```

Comment:
Do we like to differentiate platform ? For linux /usr/bin ?

Reply:
We can, but to make things easier, both mac and linux can install the binary to `/usr/local/bin` should be okay. Not sure how ClickHouse recent code addresses this

Reply:
1. There are several `usr/bin` in Install.cpp, you may like to check all of them
2. You may like to use `#if defined(OS_DARWIN)` like in this file to differentiate platforms
3. I didn't recall if /usr/local/bin is in system path or not. Probably not, but you may need validate it in all platforms; otherwise proton won't run.

Reply:
For point 1, I updated 2 other /usr/bin to /usr/local/bin. For #2, leave this to haohang or others. For #3, at least on mac /usr/local/bin is in $PATH.

For next steps
* one solution is to merge it as verify it in 1.5.5. I don't think it will make things worse
* or haohang or other Proton engineers to handle this, better to enhance homebrew script to setup the folders too
* someone can also generate binary with this branch for linux and mac. With the CI, I can only get a docker image for a branch.

Before this is merged, I will start proton server with a config file

Reply:
if local bin doesn't work for different linux distributions, it will be a regression (we fix one, the other popped up). 

@yokofly could please pick this up for complete fix
1. For linux (prod distribution), let's keep using first class path /usr/bin/
2. For MacOS, use `/usr/local/bin` moving forward.
3. Fix other installation breakage if there is any

Thanks !